ProfileManager.js

```diff
@@ -86,7 +86,7 @@ export default class ProfileManager {
     return dataHub;
   }
 
-  async createProfile({type, name}) {
+  async createProfile({type, name, color}) {
```

Comment:
We should use `{type, ...rest}` here instead and then spread `rest` into the profile doc later on down. The `color` property isn't special and other apps may want to store other things in the profile doc.

Reply:
I am not familiar with this implementation. So you are saying literally change this to `{type, ...rest}`? Do I need to specify what ...rest is on line 113?